__tests__/diastereotopicIDs.js

```diff
@@ -21,21 +21,23 @@ describe('diastereotopicIDs', () => {
       expect(diaIDsH.indexOf(diaID)).toBeGreaterThan(-1);
     }
 
-    expect(diaIDsH).toStrictEqual([
-      'gOp@DiekjjiJ@qAP_iDCaU@',
-      'gOp@DiVMjjij@qAP_iDCaU@',
-      'gOp@DiWMjj`FHJC}H`\\Jh',
-      'gOp@DjWkjj`FHJC}H`\\Jh',
-      'gOp@DfUkjj`FHJC}H`\\Jh',
-      'daD@`@fTfYVzjjdbaB@_iB@bUP',
-      'daD@`@fTfYVzjjdbaJ@_iB@bUP',
-      'daD@`@fTfUjZjjdb`b@_iB@bUP',
-      'daD@`@fTfUjZjjdb`j@_iB@bUP',
-      'daD@`@fTfUzZjjdbL`_iB@bUP',
-      'daD@`@fTfUzZjjdbM@_iB@bUP',
-      'daD@`@fTfevzjj`A~dHBIU@',
-      'daD@`@fTeeVzjj`A~dHBIU@',
-    ]);
+    expect(diaIDsH).toMatchInlineSnapshot(`
+      [
+        "gOp@DiekjjiJ@qAP_iDCaU@",
+        "gOp@DiVMjjij@qAP_iDCaU@",
+        "gOp@DiWMjj\`FHJC}H\`\\Jh",
+        "gOp@DjWkjj\`FHJC}H\`\\Jh",
+        "gOp@DfUkjj\`FHJC}H\`\\Jh",
+        "daD@\`J@DjYvzjjdBbB@_iBTbUP",
+        "daD@\`J@DjYvzjjdBbJ@_iBTbUP",
+        "daD@\`N@DjWjZjjdB\`b@_iB\\bUP",
+        "daD@\`N@DjWjZjjdB\`j@_iB\\bUP",
+        "daD@\`N@DjWzZjjdBL\`_iB\\bUP",
+        "daD@\`N@DjWzZjjdBM@_iB\\bUP",
+        "daD@\`B@LddRK]UUP@RDIDj\`",
+        "daD@\`F@DjUZzjj\`A~dHrIU@",
+      ]
+    `);
```

Comment:
@lpatiny Some ids have changed. Should we do a major release of this library? I don't know exactly what the difference is. Maybe the new ids are not compatible with older versions (how can we know?).

Reply:
If I understand correctly, the changes are an addition of 3 new query feature fields:
<img width="807" alt="image" src="https://user-images.githubusercontent.com/2352663/196024493-fee8cd06-abcf-43dc-9613-97a980ae8d24.png">

Reply:
For me it is quite complex because we store the NMR assignment as a diastereotopicID (in the .nmrium file for example). This means that changing the code force us to have a migration path and it would not be compatible with previous versions.

Reply:
What should I do?

Reply:
I created an issue:

https://github.com/Actelion/openchemlib/issues/80

I tested the new code and they are not depicted correctly with the old version (current JS version) of openchemlib. It can be tested on https://www.cheminfo.org/?viewURL=https%3A%2F%2Fcouch.cheminfo.org%2Fcheminfo-public%2F1cc9e892242664b1d5a37312bda159ef%2Fview.json&loadversion=true&fillsearch=Display+OCLcode+oclID

But to be sure I would need that you 'escape' the result because I'm not sure there are no problematic backspace and we deal correctly with escape.

Are you able to add here the list of the 'escape' of all the strings ? (the view unescape them)

Reply:
escaped version:
```
      [
        'gOp@DiekjjiJ@qAP_iDCaU@',
        'gOp@DiVMjjij@qAP_iDCaU@',
        'gOp@DiWMjj%60FHJC%7DH%60%5CJh',
        'gOp@DjWkjj%60FHJC%7DH%60%5CJh',
        'gOp@DfUkjj%60FHJC%7DH%60%5CJh',
        'daD@%60J@DjYvzjjdBbB@_iBTbUP',
        'daD@%60J@DjYvzjjdBbJ@_iBTbUP',
        'daD@%60N@DjWjZjjdB%60b@_iB%5CbUP',
        'daD@%60N@DjWjZjjdB%60j@_iB%5CbUP',
        'daD@%60N@DjWzZjjdBL%60_iB%5CbUP',
        'daD@%60N@DjWzZjjdBM@_iB%5CbUP',
        'daD@%60B@LddRK%5DUUP@%7FRDIDj%60',
        'daD@%60F@DjUZzjj%60A%7EdHrIU@'
      ]
```